src/Scripting/CSharpTest/ObjectFormatterTests.cs

```diff
@@ -943,5 +943,65 @@ public void StackTrace_Dynamic()
                 Assert.Equal(expected, actual);
             }
         }
+
+        private static class ParametersFixture
+        {
+            [MethodImpl(MethodImplOptions.NoInlining)]
+            public static void Method(ref char c, out DateTime d)
+            {
+                throw new Exception();
+            }
+
+            [MethodImpl(MethodImplOptions.NoInlining)]
+            public static void Method<U>(ref U u)
+            {
+                throw new Exception();
+            }
+        }
+
+        [Fact]
+        public void StackTrace_RefOutParameters()
+        {
+            try
+            {
+                char c = ' ';
+                DateTime date;
+                ParametersFixture.Method(ref c, out date);
+            }
+            catch (Exception e)
+            {
+                const string filePath = @"z:\Fixture.cs";
+
+                var expected =
+$@"Exception of type 'System.Exception' was thrown.
+  + Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests.ObjectFormatterTests.ParametersFixture.Method(ref char, out System.DateTime){string.Format(ScriptingResources.AtFileLine, filePath, 10155)}
+  + Microsoft.CodeAnalysis.CSharp.Scripting.Hosting.UnitTests.ObjectFormatterTests.StackTrace_RefOutParameters(){string.Format(ScriptingResources.AtFileLine, filePath, 10172)}
+";
+                var actual = s_formatter.FormatException(e);
+                Assert.Equal(expected, actual);
+            }
+        }
+
+        [Fact]
+        public void StackTrace_GenericRefParameter()
+        {
+            try
+            {
+                char c = ' ';
+                ParametersFixture.Method<char>(ref c);
+            }
+            catch (Exception e)
+            {
+                const string filePath = @"z:\Fixture.cs";
+
+                var expected =
```

Comment:
Could you add a comment?
```C#
// TODO (DevDiv #173210): Should show Fixture.Method<char>
```

The CLR should report the generic arguments but doesn't.